CHANGELOG.md

```diff
@@ -1,27 +1,222 @@
+- Project developers have default access to all databases within the project, consistent with previous versions
+- Added the role of project participant, with participants by default having no access to any databases
+- Participants can apply for permissions to access databases through tickets
+- Administrators can directly authorize participants with permissions to access databases
+- Administrators can revoke permissions to access databases from participants
+
+Notifications Messages
+
+- Project collaboration adds message notification feature
+- Supported event types include ticket approval status change, task execution status change, and task scheduling failures
+- The scope of notifications can be configured through rules, messages can be configured through templates
+- The notification channel supports configuring commonly used webhook channels, such as DingTalk, Lark, and WeCom. It also supports custom HTTP requests and setting limits on message sending
+
+System Integration
+
+- SSO supports LDAP protocol
+
+### Usability Improvements
+
+- Optimized the database selection component, standardizing the database selection interaction across product pages and adding fuzzy matching functionality for project names, data source names, and database names.
+- Added a resource tree locator key to the SQL window for quickly identifying the current database's position within the resource tree.
+- Upgraded preference settings to a top-level feature accessible directly through [Settings], new configuration options include:
+    - Whether to enable end-to-end tracing during SQL execution.
+    - Whether to continue SQL execution upon encountering errors.
+    - Customization of editor theme and font size.
+    - Configuration of editor shortcuts for running SQL commands.
+    - Setting the default workspace.
+    - Whether to enable user behavior analytics.
+    - Desktop version now supports memory size configuration through JVM parameters.
+- Added database availability indicators; the database list under projects will now show unavailable statuses and reasons.
+- Improved the initiation interaction for tickets:
+    - Support for initiating various tickets directly from the database in the resource tree.
+    - Commonly used tickets (mock data, database changes, data archiving, data cleaning, database permission application) support reinitiation with editable task parameters.
+- Enhanced database change processes to detect index change statements and automatically adjust the timeout setting (120h) to prevent index change statement execution failure due to timeout.
+- Desktop version personal settings now support custom JVM configuration with memory usage control to within 1 GB.
+- Desktop version supports exporting data files larger than 4 GB.
+- Optimized the en-US language wording of the product.
+
+### Bug fixes
+
+Connection Session
```

Comment:
Connection Sessions:
- Failure to promptly clear references after connection session expiration, leading to resource leaks and potential memory consumption increases.
- In high-frequency usage scenarios, executing SQL queries or viewing table data may encounter interface freezes and unresponsive behavior.
- After modifying the username case sensitivity in data source configuration, connecting to OceanBase Oracle may result in errors.
- Occasional occurrence of 404 errors when opening the SQL console.

Reply:
acccepted, will applay changes in PR https://github.com/oceanbase/odc/pull/2154